packages/mobile/ios/celo/Info.plist

```diff
@@ -17,7 +17,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.8.2</string>
+	<string>$(MARKETING_VERSION)</string>
```

Comment:
Just throwing it out there. It's not too late to bump everything up to 2.0.0 and save us the complexity of a separate marketing version!

Reply:
We love our 1.0.0 ... if we went with 2.0.0 then it would be too late to decide we wanted to go back to 1.x.x though 😄

Reply:
wondering is this kept when we use the script to bump the version (`pre-deploy.sh`)?